app/src/main/java/grinnell/appdev/edu/lyles/ItemAdapter.java

```diff
@@ -0,0 +1,160 @@
+        public TextView titleTextView;
+        public TextView priceTextView;
+        public ImageView imageView;
+        public ImageButton favoriteButton;
+        public TextView detailsTextView;
+
+        public ViewHolder(View itemView) {
+            super(itemView);
+            titleTextView = (TextView) itemView.findViewById(R.id.tvTitle);
+            priceTextView = (TextView) itemView.findViewById(R.id.tvPrice);
+            imageView = (ImageView) itemView.findViewById(R.id.ivFood);
+            favoriteButton = (ImageButton) itemView.findViewById(R.id.btnFavorite);
+            detailsTextView = (TextView) itemView.findViewById(R.id.tvDetails);
+
+            itemView.setOnClickListener(this);
+        }
+
+        @Override
+        public void onClick(View view) {
+            if (expandedIndex != -1) {
+                this.getAdapter().expandContractItem((ViewHolder) mRecyclerView.findViewHolderForAdapterPosition(expandedIndex));
+            }
+            this.getAdapter().expandContractItem(this);
+            expandedIndex = this.getAdapterPosition();
+        }
+
+        public ItemAdapter getAdapter() {
+            return ItemAdapter.this;
+        }
+    }
+
+    private static final int RESOURCE_ID = 0;
+    private static final String DOLLAR_SIGN = "$";
+
+    private Context mContext;
+    private ArrayList<MenuItem> mMenuItems;
+
+    private FavoritesManager mFavoritesManager;
+    private boolean mIsFavoritesTabClicked;
+
+    private int expandedIndex = -1;
+    private RecyclerView mRecyclerView;
+
+    public ItemAdapter(Context context, ArrayList<MenuItem> menuItems, boolean favTab) {
+        mContext = context;
+        mMenuItems = menuItems;
+        mFavoritesManager = new FavoritesManager(context, menuItems);
+        mIsFavoritesTabClicked = favTab;
+    }
+
+    @Override
+    public ItemAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+
+        Context context = parent.getContext();
+        LayoutInflater inflater = LayoutInflater.from(context);
+        mRecyclerView = (RecyclerView) parent;
+
+        View menuItemView = inflater.inflate(R.layout.item_user_card, parent, false);
+
+        ViewHolder viewHolder = new ViewHolder(menuItemView);
+        return viewHolder;
+    }
+
+    @Override
+    public void onBindViewHolder(final ViewHolder viewHolder, int position) {
+
+        final MenuItem menuItem = mMenuItems.get(position);
+        final int itemPosition = position;
+
+        TextView titleTextView = viewHolder.titleTextView;
+        titleTextView.setText(menuItem.getTitle());
+
+        TextView priceTextView = viewHolder.priceTextView;
+        priceTextView.setText(DOLLAR_SIGN + menuItem.getPrice());
+
+        ImageView imageView = viewHolder.imageView;
+        Glide.with(this.getContext()).load(menuItem.getImageUrl())
+                .override(300, 300).into(imageView);
```

Comment:
To make this more readable (and understandable), let's extract 300 as its own constant with a proper, descriptive name.